qa/L0_output_fields/test.sh

```diff
@@ -99,14 +99,14 @@ for CONFIG_FILE in ${LIST_OF_CONFIG_FILES[@]}; do
             RET=1
         fi
         
-        check_log_table_row_column $ANALYZER_LOG $SERVER_METRICS_NUM_COLUMNS ${#GPUS[@]} "Server\ Only:"
```

Comment:
Why did you remove `#GPUS[@]`? The test will fail if it lands on a multi gpu machine.

Reply:
It is not supposed to do a profile for this test. This test only tests the analyze portion, so it loads data from a checkpoint. Th checkpoint only has 1 measurement, for which we are adjusting the output fields